From 14d1952c17637b80923d1bfaf3b6b5f8cf032147 Mon Sep 17 00:00:00 2001
From: Emanuele Giuseppe Esposito <eesposit@redhat.com>
Date: Wed, 14 Dec 2022 09:31:51 +0100
Subject: [PATCH] Ensure network ready before cloud-init service runs on RHEL
 (#1893)

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2152100

commit 6e725f36647407d201af0603d7db11fc96a93d4d
Author: James Falcon <james.falcon@canonical.com>
Date:   Tue Dec 13 10:55:23 2022 -0600

    Ensure network ready before cloud-init service runs on RHEL (#1893)

    LP: #1998655

Signed-off-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>
---
 systemd/cloud-init.service.tmpl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/systemd/cloud-init.service.tmpl b/systemd/cloud-init.service.tmpl
index c170aef7..fc984d5c 100644
--- a/systemd/cloud-init.service.tmpl
+++ b/systemd/cloud-init.service.tmpl
@@ -16,6 +16,7 @@ After=networking.service
                   "miraclelinux", "openEuler", "rhel", "rocky", "virtuozzo"] %}
 After=network.service
 After=NetworkManager.service
+After=NetworkManager-wait-online.service
 {% endif %}
 {% if variant in ["suse"] %}
 After=wicked.service
-- 
2.38.1

